templates/out-of-sync.j2

```diff
@@ -106,9 +107,16 @@ $(document).ready(function () {
                 <span class="text-success" title="added">+{{added}}</span>&nbsp;<span class="text-danger" title="removed">-{{removed}}</span>
             </td>
             <td>{{ d.dependent_depth }}</td>
+            <td data-order="{{d.status.mathlib4_sync_prs | length}}">
+                {% set separator = joiner('<br />') %}
```

Comment:
I'm a bit suspicious that jinja is going to HTML-escape this, but it's hard to tell right now as we don't have any duplicate forward-port PRs!

Reply:
Fixed in https://github.com/leanprover-community/mathlib-port-status/commit/1fc623179ec6754015cb5fc641834fc736bbe991